src/pear.c

```diff
@@ -88,7 +88,7 @@ pear__on_thread(void *data) {
 
   pear__bootstrap_start = uv_hrtime();
 
-  err = appling_bootstrap(&loop, js, &pear__bootstrap, pear__platform.key, pear__path, pear__on_bootstrap);
+  err = appling_bootstrap(&loop, js, &pear__bootstrap, pear__platform.key, pear__path, pear__on_bootstrap, pear__app_link.id);
```

Comment:
```suggestion
  err = appling_bootstrap(&loop, js, &pear__bootstrap, pear__platform.key, pear__app_link, pear__path, pear__on_bootstrap);
```

See https://github.com/holepunchto/libappling/pull/14#pullrequestreview-3131101291.